areas-of-d-usage.dd

```diff
@@ -322,4 +322,7 @@ Macros:
                 margin-top: -200px;
             }
         }
+        h2 {
```

Comment:
Why? This is explicity targeted to add more padding on this page which highly likely isn't needed for all pages.